presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml

```diff
@@ -4,3 +4,8 @@ databases:
   presto:
     host: presto-master
     configured_hdfs_user: hive
+    cli_ldap_truststore_path: /etc/ldap/cacerts.jks
```

Comment:
That all seems like a scope for a separate PR though. So, please either revert the config merge, or separate to its own commit.